### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by yigitm**
- Get, Post, Delete async functions are added

Thank you for your time.

### CODE REVIEW

**Review by peterrobert — CHANGES_REQUESTED** [review]
### INVALID 🛑 
HI @yigitm,

Great work for submitting your project for review. However, I couldn't review your project because of the following reasons ⬇️ 

- [x] It will be great to add a professional readme file for your project. Remember this is required of you for every project you create. It seems you forgot to add it

However, don't worry just add a readme and submit your project for another review.

File: src/components/bookList.js [hunk 1]
```diff
@@ -11,15 +11,13 @@ const Booklist = () => {
   };
 
   const displayB = books.map((book) => (
```
Comment (SajjadAhmad14) [inline src/components/bookList.js:13]: The user is getting books from the server.  When the user refreshes the browser no book is shown even if the user does not remove any book. Please make sure that books do not hide when the browser is refreshed and all books are shown in the server when the user clicks on the `BOOKS` link in the navigation bar.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,86 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Book Store
 
-## Available Scripts
+## About
```
Comment (SajjadAhmad14) [inline README.md:6]: Please add the `npm start` command to README so the user is able to start

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Yiğit, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/bookList.js (hunk 1 above)
Comment (yigitm) [reply src/components/bookList.js:13]: Thank you for the review. I think there is a problem on API - server-side cause all functionality was working fine. Similar issues happened to one of my teammates as well. Thanks anyway, I will report this API issue in the next review.

File: README.md (hunk 2 above)
Comment (yigitm) [reply README.md:6]: Thank you. I add it.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @yigitm, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**